views/metabox/cpcss/container.php

```diff
@@ -16,15 +16,20 @@
 ?>
 <div class="inside">
 	<h3><?php esc_html_e( 'Critical Path CSS', 'rocket' ); ?></h3>
-	<div id="rocket-metabox-cpcss-notice"></div>
 	<div id="rocket-metabox-cpcss-content">
 	<?php do_action( 'rocket_metabox_cpcss_content' ); ?>
 	</div>
 </div>
-<?php if ( ! empty( $data['disabled_description'] ) ) : ?>
-<div class="components-notice is-notice is-warning">
+<div id="cpcss_response_notice" class="components-notice is-notice is-warning">
 	<div class="components-notice__content">
-		<p><?php echo esc_html( $data['disabled_description'] ); ?></p>
+		<?php if ( ! empty( $data['disabled_description'] ) ) : ?>
+			<p><?php echo esc_html( $data['disabled_description'] ); ?></p>
+		<?php endif; ?>
 	</div>
 </div>
-<?php endif; ?>
+<script>
+	var cpcss_rest_url       = '<?php echo esc_url( $data['cpcss_rest_url'] ); ?>';
+	var cpcss_rest_nonce     = '<?php echo esc_html( $data['cpcss_rest_nonce'] ); ?>';
+	var cpcss_generate_btn   = '<?php esc_html_e( 'Generate Specific CPCSS', 'rocket' ); ?>';
+	var cpcss_regenerate_btn = '<?php esc_html_e( 'Regenerate specific CPCSS', 'rocket' ); ?>';
```

Comment:
If these contain a `'` character, we're screwed.
```suggestion
	var cpcss_rest_url       = '<?php echo esc_js( $data['cpcss_rest_url'] ); ?>';
	var cpcss_rest_nonce     = '<?php echo esc_js( $data['cpcss_rest_nonce'] ); ?>';
	var cpcss_generate_btn   = '<?php echo esc_js( __( 'Generate Specific CPCSS', 'rocket' ) ); ?>';
	var cpcss_regenerate_btn = '<?php echo esc_js( __( 'Regenerate specific CPCSS', 'rocket' ) ); ?>';
```